indico/modules/events/registration/controllers/management/privacy.py

```diff
@@ -27,8 +27,9 @@ def _process(self):
         form = RegistrationPrivacyForm(event=self.event, regform=self.regform, visibility=[
             self.regform.publish_registrations_participants.name,
             self.regform.publish_registrations_public.name
-        ])
+        ], retention_period=self.regform.retention_period)
         if form.validate_on_submit():
+            form.populate_obj(self.regform, skip={'visibility'})
```

Comment:
hm since we just have two fields and already skip one maybe it's better to use `only=...` (or manually set `self.regform.retention_period = form.retention_period.data`)?